scripts/modules/apm_agents.py

```diff
@@ -182,6 +182,36 @@ def _content(self):
         )
 
 
+class AgentPhpApache(Service):
+    SERVICE_PORT = 8030
+
+    def __init__(self, **options):
+        super(AgentPhpApache, self).__init__(**options)
+        if options.get("enable_apm_server", True):
+            self.depends_on = {
+                "apm-server": {"condition": "service_healthy"},
+            }
+
+    @add_agent_environment([
+        ("apm_server_secret_token", "ELASTIC_APM_SECRET_TOKEN"),
+        ("apm_server_url", "ELASTIC_APM_SERVER_URL"),
```

Comment:
Does it overwrite the .ini file property? https://github.com/elastic/apm-integration-testing/pull/863/files#r501594543

Reply:
Good question. I think we can safely remove this from the ini file and expect the environment variable to always be set.

Reply:
`php.ini` has higher precedence that environment variable in case the same option is defined in both ways. I’ll make sure docs mention that.